DE.yaml

```
@@ -369,19 +372,19 @@ ITEM_SHOUTMESSAGE_NOT_DROPPABLE_HERE: Du kannst den Gegenstand hier nicht fallen
 ITEM_SHOUTMESSAGE_ONLY_IN_MINILAND: Du kannst dies nur im Mini-Land tun.
 ITEM_SHOUTMESSAGE_SAME_FACTION: Du kannst nicht in die gleiche Fraktion wechseln!
 ITEM_SHOUTMESSAGE_SAME_SP_WINGS_ALREADY_SET: Du kannst nicht die selben Flügel verwenden!
-ITEM_SHOUTMESSAGE_VEHICLE_DEBUFF: You can't change into mount because of a negative effect!
-ITEM_USAGE_MSG_ITEM_DISABLED_TEMPORARILY: This item has been disabled temporarily.
+ITEM_SHOUTMESSAGE_VEHICLE_DEBUFF:  Aufgrund eines schlechten Effekts kannst du dein Reittier nicht benutzen.

```

```suggestion
ITEM_SHOUTMESSAGE_VEHICLE_DEBUFF:  Aufgrund eines schlechten Effekts kannst du dein Reittier nicht verwenden.
```suggestion
ITEM_SHOUTMESSAGE_VEHICLE_DEBUFF:  Aufgrund eines schlechten Effekts kannst du dein Reittier nicht benutzen.

```
```